### DISCUSSION TITLE
Arnold Information no longer exported with USD

### DISCUSSION THREAD

**Discussion by cdwasden** [Autodesk/maya-usd#2722] [Q&A]
I have been working on a test script for us to generate USD files from our existing assets. I was previously using verison 0.16 of the maya-usd plugin. In that plugin, I could save the job context as Arnold so that Arnold materials and information were saved into the USD file.

Since updating to version 0.19, Arnold information is no longer being saved in the USD file on export. This is despite setting the Plug-In Configuration to Arnold in the export window:
![image](https://user-images.githubusercontent.com/3377907/201786062-b17f8151-d374-4336-8495-82cd376e6a65.png)

I'm just wondering if I missed something in between the change between versions? Is there a way for maya-usd to save Arnold information while exporting?

**Comment by tanant** [comment]
I recently was doing some (unrelated) roundtripping with the Arnold plugin config for ST maps and didn't _immediately_ see anything missing.. do you have a generic .ma you can add that i can try locally with?

**Reply by cdwasden** [reply]
Thanks for your help!

I've attached a zip with a Maya file, it contains a sphere with an aiStandardSurface material attached to it. I exported the file using the settings from the image earlier in the thread - and I've also attached the USD file that is generated from it. As you can see, no Arnold information comes along with it.
[arnold_mat_test.zip](https://github.com/Autodesk/maya-usd/files/10107622/arnold_mat_test.zip)

**Reply by tanant** [reply]
Hmm.. that looks odd - I'm just making sure I'm on stock versions of all the relevant plugins instead of dev things (and my setup is rather wacky), can you cross check what plugins you're running that side? I'm on MtoA 5.2.0  / MayaUSD 0.19.0 here for reference, and here's my full option list for reference in case something jumps out.

![image](https://user-images.githubusercontent.com/2452284/204681303-46803292-9443-47c5-a9de-4e11596ea005.png)

**Reply by cdwasden** [reply]
Thanks again.

I don't think I have any plugins out of the ordinary - my version is MtoA 5.1 and Maya-Usd 0.19.

I think all of your settings match what I have been using!

**Reply by santosd** [reply]
Hi @cdwasden. this should work as expected here are the compatibility chart: https://knowledge.autodesk.com/support/maya/learn-explore/caas/simplecontent/content/maya-usd-bifrost-and-arnold-compatibility.html
So as long as you have MtoA 5.1.2 it should work. When you create a stage from the exported file, are you able to render it in Arnold? The Viewport may not display the texture, but it should render in MtoA.

**Reply by cdwasden** [reply]
Hi David - thanks for responding. I'll try to reinstall the plugin to see if that helps resolve it.

The issue I was running into wasn't so much about rendering as much as it was that no Arnold information was being exported into the USD file, whereas in version 0.16 Arnold information was being exported. I'm not sure what the difference is between it, but I'll first try reinstalling the plugin and see if that helps resolve it.
